components/brave_wallet/common/buildflags/buildflags.gni

```diff
@@ -0,0 +1,3 @@
+declare_args() {
+  filecoin_enabled = true
```

Comment:
why are we adding a buildflag for a feature that is true on all platforms? This should be a feature flag

Reply:
Changed to feature flag. Thanks.